modules/Emails/Email.php

```diff
@@ -3383,4 +3383,208 @@ public function revertFieldNullable($field)
             unset($this->modifiedFieldDefs[$field]);
             }
     	}
+
+    /**
+     * Uses the $_REQUEST to populate the fields of an Email SugarBean.
+     *
+     * @param Email $bean
+     * @return bool|Email|SugarBean
+     */
+    public function populateBeanFromRequest(Email $bean) {
```

Comment:
Renamed signature and passed $_REQUEST into method instead. It unfortunately takes 200 lines to populate the request. When refactoring after we have autoloading we should consider using PSR-7.